randovania/data/json_data/prime1.txt

```diff
@@ -2139,10 +2291,14 @@ Asset id: 2949963832
 > Dock to Main Ventilation Shaft Section A; Heals? False
   > Main Ventilation Shaft Section A/Dock to Main Ventilation Shaft Section B
       Trivial
+  > Main Ventilation Shaft Section B/Dock to Main Ventilation Shaft Section C
+      Trivial
 
 > Dock to Main Ventilation Shaft Section C; Heals? False
   > Main Ventilation Shaft Section C/Dock to Main Ventilation Shaft Section B
       Trivial
+  > Main Ventilation Shaft Section B/Dock to Main Ventilation Shaft Section A
+      Wave Beam and Thermal Visor
 
 ----------------
 Transport to Chozo Ruins East
```

Comment:
Transport to Chozo Ruins East

Add another node to this room called "Elevator to Chozo Ruins - Transport to Tallon Overworld East" with the requirement Scan Visor.

Transport to Magmoor Caverns East

Add another node to this room called "Elevator to Magmoor Caverns - Transport to Tallon Overworld West" with the requirement Scan Visor.